pkg/graph/resolvers/resolver_test_utilities.go

```diff
@@ -25,15 +26,16 @@ import (
 
 // TestConfigs is a struct that contains all the dependencies needed to run a test
 type TestConfigs struct {
-	DBConfig  storage.DBConfig
-	LDClient  *ld.LDClient
-	Logger    *zap.Logger
-	UserInfo  *models.UserInfo
-	Store     *storage.Store
-	S3Client  *upload.S3Client
-	PubSub    *pubsub.ServicePubSub
-	Principal *authentication.ApplicationPrincipal
-	Context   context.Context
+	DBConfig      storage.DBConfig
+	LDClient      *ld.LDClient
+	Logger        *zap.Logger
+	UserInfo      *models.UserInfo
+	Store         *storage.Store
+	S3Client      *upload.S3Client
+	PubSub        *pubsub.ServicePubSub
+	Principal     *authentication.ApplicationPrincipal
+	Context       context.Context
+	FetchUserInfo func(context.Context, string) (*models.UserInfo, error)
```

Comment:
Would it be easier to just put the entire Okta client here? We do similar stuff with `Store`, and `S3Client`

Reply:
Yeah! I think that's a good idea, I'll try it out.

Reply:
I updated it! Largely throughout the test code, a stubFetchUserInfo method is being called, but I think that is ok for those cases being tested.

Reply:
Nice. Might just be me, but I like seeing `OktaClient oktaapi.Client` and just knowing that we have those methods available in the test client, rather than having to read/understand `FetchUserInfo func(context.Context, string) (*models.UserInfo, error)` every time 😂

Reply:
💯! Thanks for suggesting it! I think that will be an improvement to have that ability as well!